pkg/download/download.go

```diff
@@ -86,15 +87,15 @@ func download(client *http.Client, u *url.URL, path string, extract bool) error
 			case strings.HasSuffix(u.Path, "gz"):
 				gr, err := gzip.NewReader(resp.Body)
 				if err != nil {
-					return err
+					return fmt.Errorf("gzip reader: %w", err)
```

Comment:
sanity check here and below: would it be useful to you to know the URL of the asset that failed?

Reply:
ah, I think this is probably handled in `RetryNotify`

Reply:
yes, but it's also upstream too in the calling func